src/App.css

```diff
@@ -1,5 +1,31 @@
-#root {
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,600;1,700&family=Source+Sans+Pro:wght@300;400&display=swap');
+
+/* FONTS
+font-family: 'Playfair Display', serif;
+weight of 600 and 700
+
+font-family: 'Source Sans Pro', sans-serif;
+weight of 300 and 400
+*/
+
+/* COLORS
+orange: f4a431
+dark-orange: c65e26
+green: 366e41
+dark-green: 173e2e
+*/
+
+body {
+  background-color: #173e2e;
+  background-image: url('img/background.jpg');
   height: 100vh;
+  background-size: cover;
+  background-position: center;
+  text-align: center;
+  display: grid;
```

Comment:
Grid is one of my all-time favorite tools. Nice work using it here to center everything!

Reply:
I'm just learning grid, so I'm still experimenting around with all it can do! But I feel like it's sooo much easier than flexbow.